package.json

```diff
@@ -72,4 +72,4 @@
     ],
     "testEnvironment": "node"
   }
-}
+}
```

Comment:
Please add a new line here , so we can get rid of this red mark

Reply:
I am not able to add new line as per the lint rules. Can we ignore this?